distributed/client.py

```diff
@@ -1532,6 +1532,7 @@ def map(
         self,
         func,
         *iterables,
+        chunksize=1,
```

Comment:
Going along with the argument of "this isn't quite the same as CF chunksize, wouldn't our default argument here be `math.inf` (or perhaps None)?